test/test_project.py

```diff
@@ -132,23 +396,25 @@ def test_file_export(self):
         with self.assertRaises(ValueError):
             self.reg_proj.export_file(record=record, field='dob')
         # Delete and make sure we get an RedcapError with next export
-        self.reg_proj.delete_file(record, field)
-        with self.assertRaises(RedcapError):
-            self.reg_proj.export_file(record, field)
-
+        # self.reg_proj.delete_file(record, field)
```

Comment:
Is this related to the tests not actually making any kind of side effect on the real server anymore?

Reply:
Yes, as everything is mocked now.